packages/streamlib-moq/tests/test_processors.py

```
@@ -175,18 +202,380 @@ def test_the_relay_refusal_says_where_a_draft_16_token_goes():
 
 
 def test_a_subscriber_naming_no_track_at_all_is_refused_by_name():
-    """Two static output ports and no track named for either would subscribe to
+    """Three static output ports and no track named for any would subscribe to
     nothing and produce nothing, which reads from outside as a hang."""
-    with pytest.raises(ValueError, match="video_track"):
+    with pytest.raises(ValueError, match="video_track.*audio_track.*data_track"):
```

Ruff (0.16.3)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)